builder/parser/parser.go

```diff
@@ -61,6 +61,7 @@ func init() {
 		command.Entrypoint: parseMaybeJSON,
 		command.Expose:     parseStringsWhitespaceDelimited,
 		command.Volume:     parseMaybeJSONToList,
+		command.Arg:        parseNameOrNameVal,
```

Comment:
@mapuri I thought we agreed on allowing only one `key=value` in https://github.com/docker/docker/pull/15182#issuecomment-136451078 is it an oversight?